### ISSUE TITLE
Can't use nimble to install nim package such as nake.

### ISSUE THREAD

**Issue by danny-su** [nim-lang/nimble#1128]
<img width="460" alt="image" src="https://github.com/nim-lang/nimble/assets/12178855/9b061c66-a0fc-4e3e-b1e5-2674d8efee06">

I need to clone nake repo manually, and then use nimble to install from local repo.
<img width="567" alt="image" src="https://github.com/nim-lang/nimble/assets/12178855/f013e06a-413b-4cd4-93e9-2ff4830bac81">

**Comment by danny-su** [comment]
<img width="1003" alt="image" src="https://github.com/nim-lang/nimble/assets/12178855/13930e41-9732-4109-b06d-f5b960b74e62">

**Comment by xTrayambak** [comment]
Yes. This occurs a lot of the times for me too. This generally happens to me when I'm not using Cloudflare's DNS, which doesn't make any sense but it just kind of magically starts working fine again.

**Comment by danny-su** [comment]
This is caused by [an issue of execCmdEx](https://github.com/nim-lang/Nim/issues/23286)
